### ISSUE TITLE
Flows causing export issues

### ISSUE THREAD

**Issue by ArneMaes0** [SkylineCommunications/Low-Code-App-Editor#13]
When exporting an app that has a flow defined, the app crashed. Because it assumes all the dashboard data is of type DMADashboardQueryData, but now it can also be flows. See stacktrace:

```text
System.InvalidCastException: Unable to cast object of type 'Skyline.DataMiner.Web.Common.v1.Dashboards.DMADashboardFlowData' to type 'Skyline.DataMiner.Web.Common.v1.Dashboards.DMADashboardQueryData'.
   at Low_Code_App_Editor_1.LCA.AppVersion.<>c__DisplayClass8_0.<GetUsedScripts>b__0(DMADashboardData query)
   at Skyline.DataMiner.Net.Helper.EnumerableExtensions.ForEach[T](IEnumerable`1 value, Action`1 action)
   at Low_Code_App_Editor_1.LCA.AppVersion.GetUsedScripts()
   at Low_Code_App_Editor_1.Controllers.ExportController.AddScriptsToArchive(ZipArchive zip, App app)
   at Low_Code_App_Editor_1.Controllers.ExportController.ExportPackage(IEngine engine, IEnumerable`1 apps, ExportOptions options)
   at Low_Code_App_Editor_1.Script.<>c__DisplayClass22_0.<InitExport>b__7(Object sender, EventArgs e)
   at Skyline.DataMiner.Utils.InteractiveAutomationScript.Button.RaiseResultEvents()
   at Skyline.DataMiner.Utils.InteractiveAutomationScript.Dialog`1.RaiseResultEvents(UIResults uir)
   at Skyline.DataMiner.Utils.InteractiveAutomationScript.Dialog`1.ShowInteractive()
   at Skyline.DataMiner.Utils.InteractiveAutomationScript.InteractiveController.Run(IDialog startDialog)
   at Low_Code_App_Editor_1.Script.Run(IEngine engine)```

**Comment by ArneMaes0** [comment]
Should be fixed with this commit: [d3d4085](https://github.com/SkylineCommunications/Low-Code-App-Editor/commit/00d89170f6dccec8d0009053b14683e7e2ebf011)

**Comment by ArneMaes0** [comment]
Fix is released in version [1.0.0.17](https://github.com/SkylineCommunications/Low-Code-App-Editor/releases/tag/1.0.0.17)
